docs/intro/features.rst

```diff
@@ -184,23 +184,37 @@ on Automatic Spelling Correction for Russian:
+
+Deep Attention Matching Network (DAM)
+   Based on the work `Xiangyang Zhou, et al. "Multi-Turn Response Selection for Chatbots with Deep Attention Matching Network". Proceedings of the 56th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers). 2018 <http://aclweb.org/anthology/P18-1103>`__
+
 
 Available pre-trained models for ranking:
 
 .. table::
    :widths: auto
 
-   +-------------------+-------------------------------------------------------------------------------------+-----------------------+------------------+
-   |    Dataset        | Model config                                                                        | Validation (Recall@1) | Test1 (Recall@1) |
-   +===================+=====================================================================================+=======================+==================+
-   | `InsuranceQA V1`_ | :config:`ranking_insurance_interact <ranking/ranking_insurance_interact.json>`      |   72.0                |   72.2           |
-   +-------------------+-------------------------------------------------------------------------------------+-----------------------+------------------+
-   | `Ubuntu V2`_      |:config:`ranking_ubuntu_v2_interact <ranking/ranking_ubuntu_v2_interact.json>`       |   52.9                |   52.4           |
-   +-------------------+-------------------------------------------------------------------------------------+-----------------------+------------------+
-   | `Ubuntu V2`_      |:config:`ranking_ubuntu_v2_mt_interact <ranking/ranking_ubuntu_v2_mt_interact.json>` |   59.2                |   58.7           |
-   +-------------------+-------------------------------------------------------------------------------------+-----------------------+------------------+
+   +-------------------+----------------------------------------------------------------------------------------------------------------+-----------------------+------------------+
+   |    Dataset        | Model config                                                                                                   | Validation (Recall@1) | Test1 (Recall@1) |
+   +===================+================================================================================================================+=======================+==================+
+   | `InsuranceQA V1`_ | :config:`ranking_insurance_interact <ranking/ranking_insurance_interact.json>`                                 |   72.0                |   72.2           |
+   +-------------------+----------------------------------------------------------------------------------------------------------------+-----------------------+------------------+
+   | `Ubuntu V2`_      | :config:`ranking_ubuntu_v2_mt_word2vec_dam_interact <ranking/ranking_ubuntu_v2_mt_word2vec_dam_interact.json>` |   70.3                |   69.8           |
```

Comment:
Should we add r@2, r@5 metric values for all ranking models as well?

Reply:
I think it's worth doing. You could now fill cells for your models and leave them empty for my models.